app/seqexec-server-gs/src/main/resources/app.conf

```diff
@@ -63,7 +63,7 @@ seqexec-engine {
   ioTimeout = 4 seconds
   dhsTimeout = 24 seconds
   gpiUrl = "failover:(tcp://172.17.107.50:61616)?timeout=4000"
-  gpiGDS = "http://172.17.107.50:8888/xmlrpc"
+  gpiGDS = "http://172.17.107.50:8888"
```

Comment:
We may want to keep the endpoint in a route as the GMP could be exposing other endpoints over http. I guess we could use a different for for different endpoints though